.github/workflows/nix.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         authToken: ${{ secrets.HLS_CACHIX_AUTH_TOKEN }}
     - if: ${{ needs.pre_job.outputs.should_skip != 'true' }}
       run: nix build
-    - if: ${{ needs.pre_job.outputs.should_skip != 'true' }}
+    - if: ${{ needs.pre_job.outputs.should_skip != 'true' && github.repository_owner == 'haskell' }}
       run: nix develop --profile dev && cachix push haskell-language-server dev
-    - if: ${{ needs.pre_job.outputs.should_skip != 'true' }}
+    - if: ${{ needs.pre_job.outputs.should_skip != 'true' && github.repository_owner == 'haskell' }}
```

Comment:
mmm, this doesn't seem to work at all

Reply:
How about explicitly check whether the secret is defined or not?

Reply:
actions/runner/issues/520

Reply:
I guess https://github.com/actions/runner/issues/520#issuecomment-700579336 suggests a way?

Reply:
I'll try it later. We have to resolve this first, since the nix CI keeps failing, blocking other PRs.